tests/unit/work-items/work-items.test.ts

```
@@ -103,4 +103,66 @@ describe(!!(config.workspaceSlug && config.projectId && config.userId), "Work It
+
+    expect(Array.isArray(results)).toBe(true);
+    for (const item of results) {
+      expect(item.id).toBeDefined();
+      expect(item.name).toBeDefined();
+      expect(item.sequence_id).toBeDefined();
+      expect(item.project_id).toBeDefined();
+      expect(item.workspace_id).toBeDefined();
+    }
+  });
+
+  it("should advanced search work items with filters", async () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)